client/state/index.js

```diff
@@ -17,11 +18,12 @@ import ui from './ui/reducers';
 const reducer = combineReducers( {
 	sharing,
 	sites,
-	ui
+	ui,
+    notices
 } );
 
-export function createReduxStore() {
-	return applyMiddleware(
-		thunkMiddleware
-	)( createStore )( reducer );
-};
+const store = applyMiddleware(
+	thunkMiddleware
+)( createStore )( reducer );
+
+export default store;
```

Comment:
@aduth , @mtias 
I would really want to discuss this:

# Why did I do it like this?
TLDR: I needed it in https://github.com/Automattic/wp-calypso/pull/1393/files#diff-48b5d386d8ed23afbf806dd8b9435d7fR1 .

Notices are the first case that is really making use of actions cross-site, so here's the deal:

In order to SEND a notice, we need to do `reduxStore.dispatch` . We can:
a) use store reference from `context`, but we need to pass it down from controller to component that wants to send a notice. Sometimes, this can be deep.
b) wrap the component that wants to send a notice in `connect`. so in `mapDispatchToProps` actions will be wrapped in dispatch call.

Both of these options would involve re-writing a bazilion components and connecting EVERYTHING to redux, because everything is sending notices.

So, I used a singleton and global notice helper/factory.

I suspect, there will be a lot of instances that we need to get to the store without components and arbitraly.

# SSR
As @mtias told me, this can create issues with SSR because that would globalize someone's store and this would run server-side.

But this is EXACTLY what is happening right now. This module:
https://github.com/Automattic/wp-calypso/blob/master/client/notices/index.js

acts like a singleton for notices.

# What can we do?

I'm for singleton for a store. If it creates problems with SSR, maybe we can connect it somehow to rotate different stores between users.
But this will become a problem anyway.